File: Difficulty: Basic/Reverse a String/reverse-a-string.cpp
// User function Template for C++
class Solution {
  public:
    string reverseString(string& s) {
        // code here
        int start=0;
        int end=s.length()-1;
        while(start<=end){
            swap(s[start],s[end]);
            start++;
            end--;
        }
        return s;
        
    }
};

File: Difficulty: Basic/Reverse a String/reverse-a-string.java
// User function Template for Java
import java.util.*;

class Solution {
    public static String reverseString(String s) {
        StringBuilder res = new StringBuilder();

        for(int i = s.length() - 1; i >= 0; i--) {
            res.append(s.charAt(i));
        }

        return res.toString();
    }
}